docs/source/faq.rst

```diff
@@ -0,0 +1,114 @@
+==========================
+Frequently Asked Questions
+==========================
+
+This page contains frequently asked questions about MOTECH with short answers to them.
+
+.. contents::
+    :depth: 2
+
+Can Motech run on Windows?
+-----------------------------
+All of the components of the MOTECH are avaliable on Windows and it is possible to run the service on it,
```

Comment:
What do you mean with "the components are available on Windows"? The bundles are all the same for Windows and Linux.